pandas/core/window/rolling.py

```diff
@@ -1193,7 +1208,7 @@ def mean(self, *args, **kwargs):
         window_func = self._get_roll_func("roll_weighted_mean")
         window_func = get_weighted_roll_func(window_func)
         return self._apply(
-            window_func, center=self.center, is_weighted=True, name="mean", **kwargs
+            window_func, center=self.center, is_weighted=True, name="mean", **kwargs,
```

Comment:
dont' add commas back; we are in the process of updating the black version and these are removed on purpose